src/main.js

```diff
@@ -54,18 +55,24 @@ ErrorReporting(Vue, router);
 // Render function
 const render = (awesome) => awesome(Dashy);
 
-store.dispatch(Keys.INITIALIZE_CONFIG).then((thing) => console.log('main', thing));
-
 // Mount the app, with router, store i18n and render func
 const mount = () => new Vue({
   store, router, render, i18n,
 }).$mount('#app');
 
-// If Keycloak not enabled, then proceed straight to the app
-if (!isKeycloakEnabled()) {
-  mount();
-} else { // Keycloak is enabled, redirect to KC login page
-  initKeycloakAuth()
-    .then(() => mount())
-    .catch(() => window.location.reload());
-}
+store.dispatch(Keys.INITIALIZE_CONFIG).then((thing) => {
+  console.log('main', thing);
```

Comment:
Maybe don't need this here 😉

Reply:
Oh just realized it was me who left that there ! 😅

Reply:
Likely slept since then :-) 

I can remove the comment and push again. It was helpful with the race condition.